docs-src/concepts/what-is-asynchronous-activity-completion.md

```diff
@@ -19,11 +19,20 @@ The Temporal Client can then be used to both Heartbeat Activity Execution progre
 
 #### When to use Async Completion
 
-The intended use-case for this feature is when an external system has the final result of a computation, started by an Activity.
+The intended use case for this feature is when an external system has the final result of a computation, started by an Activity.
+However, often it works better for the Activity to notify the external system, complete normally, and then wait in the Workflow for a [Signal](/concepts/what-is-a-signal) from the external system.
 
-Consider using Asynchronous Activities instead of Signals if the external process is unreliable and might fail to send critical status updates through a Signal.
+Consider using Async Completion when
 
-Consider using [Signals](/concepts/what-is-a-signal) as an alternative to Asynchronous Activities to return data back to a Workflow Execution if there is a human in the process loop.
-The reason is that a human in the loop means multiple steps in the process.
-The first is the Activity Function that stores state in an external system and at least one other step where a human would “complete” the activity.
-If the first step fails, you want to detect that quickly and retry instead of waiting for the entire process, which could be significantly longer when humans are involved.
+- the external process is unreliable and might fail to send critical status updates through a Signal.
+- you want the external process to Heartbeat or receive Cancellation.
+
+Consider using Signals when you want to retry sooner a failure to notify the external system.
```

Comment:
I am not sure what this sentence is trying to say tbh.
Maybe a missing word? Sooner than what?